heureka/ui/src/App.js

```diff
@@ -27,6 +34,33 @@ const App = (props) => {
   w-full
   `
 
+  useLayoutEffect(() => {
+    // filterLabels are the labels shown in the filter dropdown, enabling users to filter services based on specific criteria. Default is support-group.
+    if (props.filterLabels) setLabels(props.filterLabels)
+
+    // predefined filters config
+    const predefinedFilters = [
```

Comment:
it seems like you're not using this array. In line 50 the setter is commented out. If you don't need this better delete it altogether. :)